files/en-us/web/css/anchor/index.md

```diff
@@ -110,18 +108,18 @@ The CSS {{glossary("inset properties")}} that accept an `anchor()` function as a
 
 When using an `anchor()` function inside an inset property value, the `<anchor-side>` parameter specified inside the `anchor()` function has to be compatible with the axis on which the inset property resides.
 
-In the case of physical inset properties and `<anchor-side>` values, this is relatively simple. `top: anchor(bottom)` is fine, for example, but `top: anchor(left)` is not valid. If `top: anchor(left, 50px)` were specified, the fallback value would be used, so `top` would get a computed value of `50px`. If no fallback is present, the inset property has no effect, as if it were set to `auto`.
+This means that physical `<anchor-side>` values can be used within the values of physical inset properties if the axis direction of the `<anchor-side>` is the same as the physical property's. In other words, the `top` and `bottom` sides are not valid within the `left` and `right` property values (or for those properties within the `inset` shorthand), and `left` and `right` are not valid within `top` and `bottom` property values. For example, `top: anchor(bottom)` is fine, as they are both vertical, or y-axis, values but `top: anchor(left)` is not valid, as `left` is a horizontal, or x-axis, value. If `top: anchor(left, 50px)` were specified, the fallback value would be used, so `top` would get a computed value of `50px`. If no fallback is present, the inset property behaves as if it were set to `auto`.
 
-You can also use physical inset values with logical `<anchor-side>` values. `top: anchor(start)` and `top: anchor(self-end)` work fine, for example, as those `<anchor-side>` values are relative to the inset property's relevant axis.
+You can use logical `<anchor-side>` values within both logical and physical inset properties as logical `<anchor-side>` values are relative to the inset property's relevant axis, whether the property is logical or relative. For example, `top: anchor(start)`, `top: anchor(self-end)`, `inset-block-start: anchor(end)` and `inset-inline-end: anchor(self-start)` all work fine.
```

Comment:
I opted to put logical values in a quick paragraph separate from physical with physical and physical with logical. Removing them from the two other conversations makes those conversations (esp the next paragraph) easier to parse.